diff --git a/node_modules/@daily-co/react-native-webrtc/android/build.gradle b/node_modules/@daily-co/react-native-webrtc/android/build.gradle
index 82479cf..8673a77 100644
--- a/node_modules/@daily-co/react-native-webrtc/android/build.gradle
+++ b/node_modules/@daily-co/react-native-webrtc/android/build.gradle
@@ -28,8 +28,11 @@ android {
     }
 }
 
+
+
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.github.jiangdongguo.AndroidUSBCamera:libausbc:3.3.3'
+    implementation("com.github.chenyeju295.AndroidUSBCamera:libausbc:3.3.6")
+
     api 'org.jitsi:webrtc:118.+'
 }
diff --git a/node_modules/@daily-co/react-native-webrtc/android/libs/libausbc-3.3.3.aar b/node_modules/@daily-co/react-native-webrtc/android/libs/libausbc-3.3.3.aar
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@daily-co/react-native-webrtc/android/libs/libnative-3.3.3.aar b/node_modules/@daily-co/react-native-webrtc/android/libs/libnative-3.3.3.aar
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@daily-co/react-native-webrtc/android/libs/libutils-3.3.3.aar b/node_modules/@daily-co/react-native-webrtc/android/libs/libutils-3.3.3.aar
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@daily-co/react-native-webrtc/android/libs/libuvc-3.3.3.aar b/node_modules/@daily-co/react-native-webrtc/android/libs/libuvc-3.3.3.aar
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@daily-co/react-native-webrtc/android/libs/libuvccommon-3.3.3.aar b/node_modules/@daily-co/react-native-webrtc/android/libs/libuvccommon-3.3.3.aar
new file mode 100644
index 0000000..e69de29
